marconi/marconi.cabal

```diff
@@ -131,6 +153,43 @@ executable marconi
     , directory
     , prettyprinter
 
+library marconi-test-lib
+  import:          lang
+  visibility:      public
+  hs-source-dirs:  test-lib
+  exposed-modules:
+    Gen.Marconi.Types
```

Comment:
Naming convention is similar to that of Cardno-Api, are we OK with that?

Reply:
:+1: